modules/custom/az_event/templates/az-event-calendar-add-button.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+/**
+* @file
+* Default theme implementation to display a calendar add button.
+*
+* @ingroup themeable
+*/
+#}
+{% set classes = [
+    'clear-both',
+] %}
+{% set links = calendar_links(title, date(start_date), date(end_date), FALSE, description, location) %}
+
+<div{{ attributes.addClass(classes) }}>
+  <!-- Button trigger modal -->
+  <button type="button" class="btn btn-hollow-primary btn-sm text-capitalize mt-1" data-toggle="modal" data-target="#{{ modal }}">
+    Add to Calendar
```

Comment:
<img width="286" alt="image" src="https://github.com/user-attachments/assets/2c196dc9-e6b9-4450-98dc-12405c123a40" />
<p></p>

Suggestion to increase spacing above the button and (if you want!) add a calendar icon to the button:
```suggestion
  <button type="button" class="btn btn-hollow-primary btn-sm text-capitalize mt-3" data-toggle="modal" data-target="#{{ modal }}">
    <span class="material-icons-sharp mr-2 align-bottom">calendar_month</span>Add to Calendar
```